src/chainparams.cpp

```diff
@@ -270,7 +268,7 @@ class CTestNetParams : public CChainParams {
  */
 class CRegTestParams : public CChainParams {
 public:
-    CRegTestParams() {
+    CRegTestParams(const ArgsManager& args) {
```

Comment:
```
2018-09-18 16:46:43 clang-tidy(pr=13311): src/chainparams.cpp:271:5: warning: single-argument constructors must be marked explicit to avoid unintentional implicit conversions [google-explicit-constructor]
```